parse-nessus-xml.py

```diff
@@ -53,7 +57,8 @@
     if vuln_report[key]["risk_factor"] != "None":
         affected_hosts = vuln_report[key]["hosts"]
         print(vuln_report[key]["full_description"])
-        if len(affected_hosts) > 4:
+#        print(vuln_report[key]["plugin_output"])
+        if len(affected_hosts) > 6:
```

Comment:
What's the limit on the number of affected hosts found for?

Reply:
It makes the output saner, for example:
```
Hypothetical vuln with > more 6 affected hosts, https://www.tenable.com/plugins/nessus/9999
	48 affected hosts found ...
Hypothetical vuln with less than 6 affected hosts, https://www.tenable.com/plugins/nessus/9999
	shibboleth-production-idp-0-cf-production
	shibboleth-production-idp-1-cf-production
	shibboleth-production-idp-2-cf-production
```